Ntuplizer/plugins/Ntuplizer.cc

```diff
@@ -245,7 +244,11 @@ Ntuplizer::~Ntuplizer()
 void Ntuplizer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup){
   
   nBranches_->reset();
-      
+
+  edm::Handle<reco::VertexCollection> vertices;
+  iEvent.getByToken(vtxToken_, vertices);
+  if( vertices->empty() ) return; // skip the event if no PV found
```

Comment:
if we just skip the event, we will have trouble when synchronising with other groups. I'd instead use the first vertex in this case.